src/api/app/components/workflow_run_detail_component.rb

```diff
@@ -22,4 +23,18 @@ def parse_payload(workflow_run)
   rescue JSON::ParserError
     workflow_run.request_payload
   end
+
+  # Old workflow run entries didn't store the configuration-related information
+  def workflow_configuration_data
+    configuration_source = [workflow_run.workflow_configuration_url,
+                            workflow_run.workflow_configuration_path].filter_map(&:presence).first
```

Comment:
Also, this part could be moved into `WorkflowRun`.

Reply:
pushed